nautobot/utilities/forms/fields.py

```diff
@@ -194,6 +194,13 @@ class CSVContentTypeField(CSVModelChoiceField):
     STATIC_CHOICES = True
 
     def prepare_value(self, value):
+        if value is None:
```

Comment:
This was required to allow this field to support `app_label.model` style, null values, or PK-based lookups depending on how the field is used.